source/operations/concepts.rst

```diff
@@ -186,7 +186,12 @@ Some common reasons for bit rot include:
 - accidental overwrites
 
 MinIO uses a hashing algorithm to confirm the integrity of an object.
-If an object becomes corrupted by bit rot, MinIO can automatically heals the object depending on the system topology and availability.
+This algorithm automatically applies at the time of any ``GET`` request that calls an object.
+If an object becomes corrupted by bit rot, MinIO can automatically :ref:`heal <minio-concepts-healing>` the object depending on the availability of parity shards for the object.
+
+While bit rot healing can be done during the routine checks of the :ref:`MinIO Scanner <minio-concepts-scanner>`, this function is off by default.
+Bit rot is a low risk, especially if an object has multiple shards spread across multiple disks.
```

Comment:
Hm. Somewhat subjective and begs the question of 'how many shards makes the risk low'.

I suggest dropping or pulling this back to

"Active bitrot healing during scanner has a high performance impact, in comparison to the low probability of bitrot affecting multiple object shards distributed across multiple drives and nodes.

Reply:
I'll take that wording. WIth `bit rot` instead of `bitrot`.